php/add-subscriber-to-list.php

```diff
@@ -0,0 +1,93 @@
+<?php
+  /**
+   * Add a 'member' to a 'list' via mailchimp API v3.x
+   * @ http://developer.mailchimp.com/documentation/mailchimp/reference/lists/members/#create-post_lists_list_id_members
+   *
+   * ================
+   * BACKGROUND
+   * Typical use case is that this code would get run by an .ajax() jQuery call or possibly a form action
+   * The live data you need will get transferred via the global $_POST variable
+   * That data must be put into an array with keys that match the mailchimp endpoints, check the above link for those
+   * You also need to include your API key and list ID for this to work.
+   * You'll just have to go get those and type them in here, see README.md
+   * ================
+   */
+
+  /**
+   * ================
+   * Sets API Key and list ID to add a subscriber
+   * ================
+   */
+  define("api_key", "your-api-key-here");
+  define("list_id", "your-list-id-here");
+
+  /**
+   * ================
+   * DESTINATION URL
+   * Note: your API URL has a location subdomain at the front of the URL string
+   * It can vary depending on where you are in the world
+   * To determine yours, check the last 3 digits of your API key
+   * ================
+   */
+  $url = 'https://us5.api.mailchimp.com/3.0/lists/' . list_id . '/members/';
+
+  /**
+   * ================
+   * DATA SETUP
+   * Encode data into a format that the add subscriber mailchimp end point is looking for
+   * Must include 'email_address' and 'status'
+   * Statuses: pending = they get an email; subscribed = they don't get an email
+   * Custom fields go into the 'merge_fields' as another array
+   * More here: http://developer.mailchimp.com/documentation/mailchimp/reference/lists/members/#create-post_lists_list_id_members
+   * ================
+   */
+  $pfb_data = array(
+    'email_address' => $_POST['emailname'],
```

Comment:
I would consider using `$_POST['email']` because 1) I'm not sure what `emailname` is, and 2) [it's a good practice](https://developers.google.com/web/updates/2015/06/checkout-faster-with-autofill#email) to have your email input with the name="email" for autofilling purposes